statsmodels/base/model.py

```diff
@@ -1257,7 +1257,10 @@ def tvalues(self):
         """
         Return the t-statistic for a given parameter estimate.
         """
-        return self.params / self.bse
+        with np.errstate(divide="ignore"):
```

Comment:
Should be shown to end users.  Probably don't suppress here or in tests.

Reply:
Will revert here; merits discussion elsewhere.